packaging/suse/supportutils-plugin-trento/trento

```diff
@@ -0,0 +1,260 @@
+	shift
+	ENTRY_LABEL="$@"
+	case $ENTRY_TYPE in
+	command) ENTRY_HEADER="#==[ Command ]======================================#" ;;
+	conf) ENTRY_HEADER="#==[ Configuration File ]===========================#";;
+	log) ENTRY_HEADER="#==[ Log File ]=====================================#" ;;
+	note) ENTRY_HEADER="#==[ Note ]=========================================#" ;;
+	summary) ENTRY_HEADER="#==[ Summary ]======================================#" ;;
+	*) ENTRY_HEADER="#==[ Entry ]========================================#";;
+	esac
+	echo "$ENTRY_HEADER" >> $LOGFILE
+	echo "# $ENTRY_LABEL" >> $LOGFILE
+	echo >> $LOGFILE
+}
+
+# Input: logfilename "text"
+log_write() {
+	LOGFILE=$LOG/$1
+	shift
+	echo "$@" >> $LOGFILE
+}
+
+_log_cmd() {
+	EXIT_STATUS=0
+	shift
+	CMDLINE=($@)
+	CMDBIN=${CMDLINE[0]}
+	CMD=($(command -v $CMDBIN))
+	echo "#==[ Command ]======================================#"
+	if [[ -x "${CMD[0]}" ]]; then
+		CMDLINE[0]="${CMD[0]}"
+		echo "# ${CMDLINE[@]}"
+		bash -c "${CMDLINE[*]}" 2>&1
+		EXIT_STATUS=$?
+	else
+		echo "# ${CMDLINE[@]}"
+		echo "ERROR: Command not found or not executable"
+		EXIT_STATUS=1
+	fi
+	echo
+	return $EXIT_STATUS
+}
+
+log_cmd() {
+	LOGFILE=$LOG/$1
+	_log_cmd "$@" >> $LOGFILE
+}
+
+
+fi
+### END Check if executed by supportconfig ###
+
+
+### TODO ###
+# Detect and use usernames/database names/vhost name from config
+#check firewall
+#run standalone if not executed by supportconfig
+# dump-scenario
+# Use GNUGPL2 license to match supportutils, instead of Apache-2.0?
+
+
+
+
+#TODO sanitize these values
+# password from AMQP_URL
+# password from DATABASE_URL
+# password from EVENTSTORE_URL
+# postgresql-password
+# postgresql-postgres-password
+# .secretKeyRef
+# adminUser
+# admin-user
+# admin-password
+# SMTP_PASSWORD
+# ADMIN_USER
+# ADMIN_PASSWORD
+# SECRET_KEY_BASE
+# ACCESS_TOKEN_ENC_SECRET
+# REFRESH_TOKEN_ENC_SECRET
+_sanitize_file_trento() {
+	CLEAN_FILE=${LOG}/$1
+	REPLACED='*REMOVED BY SUPPORTCONFIG*'
+	sed -i -e "\
+	s/\(ADMIN_PASSWORD[[:space:]]*=\).*/\1 $REPLACED/g;\
+	s/\(SECRET_KEY_BASE[[:space:]]*=\).*/\1 $REPLACED/g;\
+	s/\(ACCESS_TOKEN_ENC_SECRET[[:space:]]*=\).*/\1 $REPLACED/g;\
+	s/\(REFRESH_TOKEN_ENC_SECRET[[:space:]]*=\).*/\1 $REPLACED/g;\
+	s/\(SMTP_PASSWORD[[:space:]]*=\).*/\1 $REPLACED/g" \
+	$CLEAN_FILE
+}
+
+
+
+## Docs  ##
+log_entry $OF section "Current Documentation"
+echo -e "\
+https://documentation.suse.com/sles-sap/trento/html/SLES-SAP-trento/
+https://www.trento-project.io/web/
+https://www.trento-project.io/wanda/
+https://github.com/trento-project/agent/blob/main/README.md
+"
+#############################################################
+log_entry $OF section "Supportconfig Plugin for $TITLE v${SVER}"
+
+## Check for different Trento deployment types ##
+
+### Trento Server - Kubernetes ###
+#TODO
+if rpm -q --quiet trento-server-installer; then
+    command -v trento-support  &>/dev/null && \
+    trento-support --output stdout --collect all --output stdout
+fi
+
```

Comment:
@Thr3d could you please help me with changing this if block to something else? `trento-server-installer` is now deprecated and its presence as a package shouldn't be used to determine whether or not we have a kubernetes setup.